config/routes.rb

```diff
@@ -12,6 +12,10 @@
   get '/api/openid_connect/userinfo' => 'openid_connect/user_info#show'
   post '/api/risc/security_events' => 'risc/security_events#create'
 
+  namespace :api do
+    post '/verify/complete' => 'verify/complete#create'
+  end
```

Comment:
We could, but no one knows the route ap/verify/complete.  the post request would fail anyway with out a jwt token.

Reply:
Yep, if it's not intended to be used, it should be feature flagged out. If there's a bug in it that causes a 500 or something similar and it gets called enough, it could trigger alerts and page people.